posthog/tasks/usage_report.py

```diff
@@ -460,10 +460,11 @@ def get_teams_with_event_count_with_groups_in_period(begin: datetime, end: datet
 @timed_log()
 @retry(tries=QUERY_RETRIES, delay=QUERY_RETRY_DELAY, backoff=QUERY_RETRY_BACKOFF)
 def get_all_event_metrics_in_period(begin: datetime, end: datetime) -> dict[str, list[tuple[int, int]]]:
-    materialized_columns = get_enabled_materialized_columns("events")
-
     # Check if $lib is materialized
-    lib_expression = materialized_columns.get(("$lib", "properties"), "JSONExtractString(properties, '$lib')")
+    lib_materialized_column = get_materialized_column_for_property("events", "properties", "$lib")
+    lib_expression = (
+        lib_materialized_column.name if lib_materialized_column is not None else "JSONExtractString(properties, '$lib')"
+    )
```

Comment:
use that `get_property_string_expr` method here is the only feedback I have

Reply:
FYI, this is slightly different than the original, but not in a way that I think really matters all that much: 422d678ab4811524795abfe2e8c1c37d3e8d7bd1